mk_verificator/tests/system/test_ntp_sync.py

```diff
@@ -1,88 +1,43 @@
-import salt.client as client
-import texttable as tt
+def test_ntp_sync(local_salt_client):
+
+    data = {}
+    node_times_list = []
+    h = 0
```

Comment:
I will do that in next pull request as it is right now already very big. Can we merge that without those changes?